app/src/main/java/com/infomaniak/mail/data/api/ApiRepository.kt

```diff
@@ -47,6 +47,7 @@ object ApiRepository : ApiRepositoryCore() {
         method: ApiController.ApiMethod,
         body: Any? = null,
         okHttpClient: OkHttpClient = HttpClient.okHttpClient,
+        throwExceptions: Boolean = false
```

Comment:
This is unused, is it on purpose?

Reply:
I guess it was supposed to be passed down to the `Core.ApiController.callApi()` function.
I did it.